integration_tests/build_integration_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+. ./config.sh
+
+function setUp()
+{
+  # Builds or updates image
+  podman build -f ./Containerfile -t "$IMAGE" ../.
```

Comment:
https://kworkflow.org/content/codingstyle.html#avoid-short-options

Reply:
I think the build phase should be inside the run_integration_tests in the preparation phase.